### ISSUE TITLE
Files are saved with the .mp3 extension but are AAC

### ISSUE THREAD

**Issue by proideo** [SimpleMobileTools/Simple-Voice-Recorder#1]
I have tried the new app and I have seen that the files are recorded with the .mp3 extension but MediaInfo says that they are in AAC format.
I also notice that the sound is a bit metallic, is there any noise reduction applied to the recordings?
Thanks for this new app.

**Comment by tibbi** [comment]
Im currently using the workaround from https://stackoverflow.com/a/33054794/1967672 , as I just couldnt find a way of converting the audio files to mp3. It should work just fine in most cases though.

**Comment by tibbi** [comment]
Im not adding any noise reduction, will see how could the quality be improved

**Comment by proideo** [comment]
> Im currently using the workaround from https://stackoverflow.com/a/33054794/1967672 , as I just couldnt find a way of converting the audio files to mp3. It should work just fine in most cases though.

Yes, it plays well on the device even with the wrong extension. Perhaps, it is better to keep the AAC format since it is a more efficient codec.

**Comment by thousandsofthem** [comment]
AAC is vastly better than mp3. Voting to keep aac and change visible extension to reflect actual file type

**Comment by cacack** [comment]
I def second the AAC > mp3, but additionally would like the use of an open standard like OGG Vorbis or Opus. I'll probably make a separate issue to request that feature.

**Comment by lingnand** [comment]
I'm not sure naming these things as 'mp3' is a good idea... I'm grateful for this as a replacement of the stock voice recorder on my phone, but initially this gave me the wrong hope when I saw it save files as 'mp3'. As it turned out, the saved files could not be used in any of my media applications on PC i.e., foobar2000 or audio editing software. That gave me a huge scare, until I saw this thread and then renamed the files to 'm4a' - ah, I'm glad I haven't lost my recordings!

In short, please just call these things 'm4a' until they are proper 'mp3'.

**Comment by thousandsofthem** [comment]
❤️ 💜 💚
